### ISSUE TITLE
Doesn't work with multi-device beta

### ISSUE THREAD

**Issue by deanylev** [vasani-arpit/WBOT#256]
**Describe the bug**
WBOT does not work with the new WhatsApp multi device beta.

**To Reproduce**
1. Open WhatsApp
2. Click on Linked Devices
3. Click Multi-Device Beta and join
4. Scan the QR to log in to WBOT as normal

**Expected behavior**
The WBOT JavaScript is injected and works as normal.

**Additional context**
Once I left the multi-device beta and logged in again, WBOT worked as normal.

**Comment by vasani-arpit** [comment]
Yes. several things. I don't have an extra device in which I can test this multi-device beta. also as this is in beta they keep changing the underlying things. so I suggest we should wait until it is generally available.

**Comment by deanylev** [comment]
Makes sense to me 👍
